packages/graphiql-plugin-code-exporter/vite.config.ts

```diff
@@ -0,0 +1,31 @@
+import { defineConfig } from 'vite';
+import react from '@vitejs/plugin-react';
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [react()],
+  build: {
+    lib: {
+      entry: 'src/index.tsx',
+      fileName: 'graphiql-plugin-code-exporter',
+      name: 'GraphiQLPluginCodeExporter',
+      formats: ['cjs', 'es', 'umd'],
+    },
+    rollupOptions: {
+      external: ['@graphiql/react', 'graphql', 'react', 'react-dom'],
+      output: {
+        chunkFileNames: '[name].[format].js',
+        globals: {
+          '@graphiql/react': 'GraphiQL.React',
+          graphql: 'GraphiQL.GraphQL',
+          react: 'React',
+          'react-dom': 'ReactDOM',
+        },
+      },
+    },
```

Comment:
Also here, we don't need to handle `@graphiql/react` as we only import a type from it.
```suggestion
    rollupOptions: {
      external: ['graphql', 'react', 'react-dom'],
      output: {
        chunkFileNames: '[name].[format].js',
        globals: {
          graphql: 'GraphiQL.GraphQL',
          react: 'React',
          'react-dom': 'ReactDOM',
        },
      },
    },
```